instructure/lti_originality_report_example
# Create report from UI

_Closes PLAT-2304_
Test Plan
- In the submission index UI fetch a submission to get its attachment data.
- Verify after refreshing that a 'create report' button appears.
- Enter a score for the originality report and click the button.
- Verify an originality report is created in Canvas and the tool.
